core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review.component.html

```diff
@@ -18,34 +18,38 @@ <h3 class="oppia-mobile-question-header" title="{{questionHeader}}">
     <span aria-hidden="true">&times;</span>
   </button>
 </div>
-
 <div class="modal-body">
-  <div class="oppia-question-details" tabindex="0">
-    <strong class="oppia-difficulty-title">
-      Selected Difficulty: {{skillDifficultyLabel}}
-    </strong>
-    <div *ngIf="skillRubricExplanations.length > 0">
-      <strong class="oppia-skill-rubrics"
-              title="Use these notes to make sure your question is at the right difficulty.">
-        Notes from Skill Rubric
-      </strong>
-      <ul>
-        <li *ngFor="let explanation of skillRubricExplanations">
-          <span class="oppia-skill-explanation" [innerHtml]="explanation"></span>
-        </li>
-      </ul>
+  <section class="oppia-suggestion-review-container">
+    <div class="oppia-question-details questionSuggestionDiv" tabindex="0">
```

Comment:
Please follow existing styles for class names. (But why do you even need questionSuggestionDiv in the first place?)